With dependency I mean something like `curl` or `sha256sum` and no, manual error handling would be needed there (so a check if each variable is defined and if not `$(error message)`)
e.g. it looks like
```
root@b770552f0e7c:/modem_zip_generator# make release
Makefile:88: warning: overriding recipe for target 'updates/19.02.1-sibon-dc48370a-ota.zip'
Makefile:82: warning: ignoring old recipe for target 'updates/19.02.1-sibon-dc48370a-ota.zip'
Downloading 19.02.1-sibon-dc48370a-ota.zip...
mkdir: created directory 'updates'
/bin/bash: progress-bar: command not found
make: [Makefile:90: updates/19.02.1-sibon-dc48370a-ota.zip] Error 127 (ignored)
/usr/bin/sha256sum: updates/19.02.1-sibon-dc48370a-ota.zip: No such file or directory
updates/19.02.1-sibon-dc48370a-ota.zip: FAILED open or read
/usr/bin/sha256sum: WARNING: 1 listed file could not be read
Unpacking firmware files...
/bin/bash: line 1: j: command not found
make: [Makefile:105: src/firmware-update/] Error 127 (ignored)
Unpacking Edify interpreter...
/bin/bash: line 1: j: command not found
make: [Makefile:96: src/META-INF/com/google/android/update-binary] Error 127 (ignored)
/usr/bin/cp: cannot stat './src/firmware-update/': No such file or directory
make: *** [Makefile:64: build/modem.zip] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -73,7 +73,7 @@ $(FLASHABLEZIP): $(FIRMWARE_DIR) $(EDIFY_BINARY) $(EDIFY_SCRIPT)
 	@echo "Building flashable ZIP..."
 	@mkdir -pv "$(@D)"
 	@rm -f "$@"
-	@cd "$(TEMP_DIR)" && $(FIND) -type f | $(SORT) | $(ZIP) -X \
+	@cd "$(TEMP_DIR)" && $(FIND) -type f | LC_ALL=C $(SORT) | $(ZIP) -X \
 		"$(ROOT)/$@" -@
 	@rm -rf "$(TEMP_DIR)"
 	@echo "Result: $@"

